spec/the_google/calendar_test.rb

```diff
@@ -201,6 +201,21 @@
                 .returns api_result
         end
 
+        describe 'and the events do not have start or end values' do
+
+          let(:items) do
+            [
+              Struct.new(:summary, :start, :end).new(random_string, nil, nil),
+              Struct.new(:summary, :start, :end).new(random_string, nil, nil),
```

Comment:
It is strange.  Just FYI if you're gonna do anything with this issue - I
only saw a nil start date. I checked for nil on the end date too here, but
I didn't take the time to see if end date had a value or not. I bet it
didn't.

On Friday, March 4, 2016, Darren Cauthon <notifications@github.com> wrote:

> In spec/the_google/calendar_test.rb
> <https://github.com/darrencauthon/the_google/pull/2#discussion_r55040774>:
>
> > @@ -201,6 +201,21 @@
> >                  .returns api_result
> >          end
> >
> > +        describe 'and the events do not have start or end values' do
> > +
> > +          let(:items) do
> > +            [
> > +              Struct.new(:summary, :start, :end).new(random_string, nil, nil),
> > +              Struct.new(:summary, :start, :end).new(random_string, nil, nil),
>
> In these cases, I think it's better to write the tests for the nil start
> and the nil end. Isolate the test scenarios so you know both are covered.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/darrencauthon/the_google/pull/2/files#r55040774>.
>